addons/rain2d/rain_area.gd

```diff
@@ -233,8 +233,8 @@ func shapepos( d ):
 	if Passive:
 		return
 	
-	var mat = Matrix32( shape_transform )
-	mat.o += d.pos + get_pos()
+	var mat = Transform2D( shape_transform )
+	mat.origin += d.pos + position
```

Comment:
The changes in these files were the only ones needed for everything to work in 3.1. I think these changes should still work in 3.0, as well.